widgets/blueprints/src/BlueprintsCatalog.tsx

```diff
@@ -1,8 +1,11 @@
 import type { ReactElement } from 'react';
+import { each } from 'lodash';
 
 import BlueprintState from './BlueprintState';
 import type { BlueprintsViewProps } from './types';
 
+const translateBlueprintsButtons = Stage.Utils.getT('widgets.blueprints.buttons');
```

Comment:
Optional:

```suggestion
import { translateBlueprints } from 'widget.utils';
const translateBlueprintsButtons = Stage.Utils.composeT(translateBlueprints, 'buttons');
```

Ref.: p. 7 from [UI Translation Files Guidelines](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/1875083446/UI+Translation+Files+Guidelines)

Reply:
https://github.com/cloudify-cosmo/cloudify-stage/pull/2413/commits/fa9c4814d373d23cc0625efdbc52cdd1162bdc67 done ✔️